components/bluetooth/lib.rs

```diff
@@ -664,10 +671,12 @@ impl BluetoothManager {
         }
     }
 
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattserver-getprimaryservice
```

Comment:
Shouldn't this be https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren or at least reference it?